moleculemaker/chemscraper-frontend
# fix: workaround for error in getByJobTypeAndJobIdAndRunId

## Problem
Frontend has trouble fetching job results after adding Kubernetes job support

## Approach
* fix: temporary workaround to use `getByJobTypeAndJobId` instead - we can simply assume a single run for now

## How to Test
Test this with https://github.com/moleculemaker/mmli-backend/pull/43

Currently deployed on prod: https://chemscraper.platform.moleculemaker.org

1. Navigate to https://chemscraper.platform.moleculemaker.org/configuration
2. Submit a real (known-good) PDF for analysis
3. Wait for job to complete
    * You should eventually see the results load and display
